tests/test_inner.py

```diff
@@ -807,6 +867,9 @@ def test_source_not_removed_for_exit_plugins():
                                                   'args': {
                                                       'watcher': watch_exit,
                                                   }}],
+
+                                   buildstep_plugins=[{'name': 'build_watched'
+                                                       }],
```

Comment:
Could make this indentation better -- either all on one line, or with `'name': ...` on its own line.